utils/notionTypeParser/manualsPreviewParser.js

```diff
@@ -0,0 +1,31 @@
+import tp from '../typograf/typograf.config'
+
+import HIDDEN_MANUALS from '../../consts/hidden-manuals'
+
+function parseManualsPreview(tree) {
+    // TODO Remove hard-code after back-end feature https://github.com/ekaterinburgdev/guides-api/issues/10
+    const manualsVisible = tree.children.filter((manual) => {
+        return !HIDDEN_MANUALS.includes(manual?.properties?.pageUrl?.url)
+    })
+
+    return manualsVisible
+        .sort((a, b) => {
+            const orderA = a.properties?.order?.number
+            const orderB = b.properties?.order?.number
+            return orderB - orderA
+        })
+        .map(({ properties }) => {
+            return {
+                title: tp.execute(properties?.Name?.title[0]?.text?.content || ''),
```

Comment:
Strange field in upper case (others in lower case)

Reply:
:(

![image](https://user-images.githubusercontent.com/22644149/205453822-a0b9a5e1-c2ff-4669-a304-9d36cab649d5.png)

Reply:
I would fix it, but I'm afraid to break the back-end application